.github/workflows/ghex.yml

```diff
@@ -0,0 +1,32 @@
+name: ghex
+
+on:
+  push:
+  pull_request:
+    branches:
+      - main
```

Comment:
I suppose the long term plan is to have these workflows in the ghex/oomph/hwmalloc repos instead, but I'm just wondering if it makes sense to have the workflows here run daily or weekly as well? Otherwise they'd only be triggered on PRs, which I expect to be relatively rare?

Reply:
Good point. Yes that's the plan. Until then, I will enable weekly runs, though.